apps/docs/src/app/core/component-docs/list/list-docs.component.html

```diff
@@ -20,8 +82,8 @@
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'iconList'" [componentName]="'list'">
-    List with Icons
+<fd-docs-section-title id="icon-list" componentName="list">
+    List with icons
 </fd-docs-section-title>
 <description>It is possible to put icons on both sides.</description>
```

Comment:
"List can display icons on both sides of the list items."

Is that what you meant?